.gitignore

```diff
@@ -1,5 +1,6 @@
 .depend.*
 GNUmakefile*
+makefile_name.local
```

Comment:
Something is wrong if `make` is writing this file.  It should be a "phony" target (a makefile goal that doesn't represent a file).

Reply:
I was using `make --touch` to avoid having to partial rebuild of OpenDDS after changing `opendds_idl`. It worked, but it created a bunch of these files. I can see now that you're right, it's not marked as `PHONY:` in `rules.local.GNU`, so it's getting created when using `make --touch`. I will submit a ACE/TAO PR for this.

Reply:
Done: https://github.com/DOCGroup/ACE_TAO/pull/975

Reply:
Done: https://github.com/DOCGroup/ACE_TAO/pull/975